src/app/app.component.html

```diff
@@ -1,34 +1,39 @@
       <div class="flex flex-wrap items-center justify-between max-w-screen-xl px-4 mx-auto">
         <a routerLink="/" class="flex items-center">
-          <img ngSrc="/assets/img/logo.png" class="h-6 mr-3 sm:h-9 w-auto" alt="{{'logo.alt' | transloco}}" height="455" width="393" priority/>
+          <img ngSrc="./assets/img/logo.png" class="h-6 mr-3 sm:h-9 w-auto" alt="{{'logo.alt' | transloco}}" height="455" width="393" priority/>
           <span class="self-center text-xl font-semibold whitespace-nowrap dark:text-white">{{'app_title' | transloco}}</span>
         </a>
         <div class="flex items-end lg:order-2">
           <button data-collapse-toggle="mobile-menu-2" type="button"
                   class="inline-flex items-center p-2 ml-1 text-sm text-gray-500 rounded-lg lg:hidden hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600"
-                  aria-controls="mobile-menu-2" aria-expanded="false">
+                  aria-controls="mobile-menu-2" aria-expanded="false"
+                  (click)="toggleMobileMenu(); $event.stopPropagation()"
+                  >
             <span class="sr-only">{{'open_main_menu' | transloco}}</span>
-            <inline-svg href="/assets/img/open-menu.svg" />
-            <inline-svg href="/assets/img/close-menu.svg" />
+            <inline-svg href="assets/img/open-menu.svg" />
+            <inline-svg href="assets/img/close-menu.svg" />
           </button>
         </div>
-        <div class="items-end justify-between hidden w-full lg:flex lg:w-auto lg:order-1" id="mobile-menu-2">
-          <ul class="flex flex-col mt-4 font-medium lg:flex-row lg:space-x-8 lg:mt-0">
+        <div class="items-end justify-between hidden w-full lg:flex lg:w-auto lg:order-1" id="mobile-menu-2" [class.hidden]="!showMobileMenu" (click)="$event.stopPropagation()">
+          <ul class="flex flex-col mt-4 font-medium border-b border-gray-300 lg:border-0 lg:flex-row lg:space-x-8 lg:mt-0 bg-gray-900">
             <li>
-              <a routerLink="/" class="block py-2 pl-3 pr-4 text-white bg-purple-700 rounded lg:bg-transparent lg:text-purple-700 lg:p-0 dark:text-white" aria-current="page">{{'home' | transloco}}</a>
+              <a routerLink="/" class="block py-2 pl-3 pr-4 text-gray-500 bg-gray-400 rounded lg:bg-transparent lg:text-gray-500 lg:p-0 dark:text-white" aria-current="page">{{'home' | transloco}}</a>
             </li>
             <li>
-              <a routerLink="/register" class="block py-2 pl-3 pr-4 text-white bg-purple-700 rounded lg:bg-transparent lg:text-purple-700 lg:p-0 dark:text-white" aria-current="page">{{'register_account' | transloco}}</a>
+              <a href="/register" class="block py-2 pl-3 pr-4 text-gray-500 bg-gray-400 rounded lg:bg-transparent lg:text-gray-500 lg:p-0 dark:text-white" aria-current="page">{{'register_account' | transloco}}</a>
```

Comment:
Why this? We're not implementing registration before releasing it?

Reply:
Correct, we'll resolve this another time.